stdlib/3/collections/__init__.pyi

```diff
@@ -8,7 +8,7 @@ from typing import (
 # These are exported.
 from . import abc
 
-from typing import (
+from _collections_abc import (
```

Comment:
Can this just be `from _collections_abc import *`? I guess we should maybe guard it with a `if sys.version_info < (3, 10):` :-)

Reply:
Yeah, it can be. The implementation in collection forwards everything in `__all__`

Reply:
Upon trying to do this, pytype and mypy have started choking on import from collections / collections.abc. Even before, pytype chokes on MutableMapping (and only that one) for a reason I don't know

Reply:
What's the mypy error?

Reply:
```
running mypy --python-version 3.8 --strict-optional --no-site-packages --show-traceback --no-implicit-optional --disallow-any-generics --disallow-subclassing-any # with 957 files
third_party\2and3\requests\packages\urllib3\_collections.pyi:2: error: Module 'collections.abc' has no attribute 'MutableMapping'
third_party\2and3\requests\packages\urllib3\_collections.pyi:11: error: Class cannot subclass value of type 'Any'
third_party\2and3\requests\packages\urllib3\_collections.pyi:24: error: Class cannot subclass value of type 'Any'
stdlib\3\shelve.pyi:5: error: Name 'collections.abc.MutableMapping' is not defined
stdlib\3\shelve.pyi:5: error: Class cannot subclass value of type 'Any'
third_party\2and3\markupsafe\__init__.pyi:4: error: Module 'collections' has no attribute 'Mapping'
```

Repeated the same for each of the versions. Changing the imports to `from collections.abc` as relevant doesn't fix the issue.

Reply:
Solved the error, it's the same as the issue in #3923 with __all__